lib/jeckle/rest_actions.rb

```diff
@@ -13,11 +13,11 @@ def find(id)
       end
 
       def search(params = {})
-        response   = run_request(resource_name, params).response.body || []
+        response   = run_request(params.delete[:resource_name] || resource_name, params).response.body || []
```

Comment:
Yes I know it. In my scenario I have a backend service API which defaults to `resource_name` but in some cases the same backend service API needs a '.json' at the end of the same URL. 
I have a model setup with `resource_name` as default and sometimes I need `resource_name.json` for the same model.

We may think about a more elegant solution, but the suggested flexibility will solve this scenario for now.